lua/neogit/popups/tag/init.lua

```diff
@@ -0,0 +1,27 @@
+local actions = require("neogit.popups.tag.actions")
+local popup = require("neogit.lib.popup")
+
+local M = {}
+
+function M.create()
+  local p = popup
+    .builder()
+    :name("NeogitTagPopup")
+    :arg_heading("Arguments")
+    :switch("f", "force", "Force")
+    :switch("a", "annotate", "Annotate")
+    :switch("s", "sign", "Sign")
+    :switch("u", "local-user", "Sign as", { enabled = false })
```

Comment:
As this takes user input, it's an `option` not a `switch`. Switch is just for boolean flags.
```suggestion
    :option("u", "local-user", "", "Sign as", { key_prefix = "-" })
```